pkg/apis/cluster/v1alpha1/types.go

```diff
@@ -116,6 +137,66 @@ type ClusterSpec struct {
 	// any resource that does not tolerate the Taint.
 	// +optional
 	Taints []corev1.Taint `json:"taints,omitempty"`
+
+	// ResourceModels is the list of resource modeling in this cluster. Each modeling quota can be customized by the user.
+	// Modeling name must be one of the following: cpu, memory, storage, ephemeral-storage.
+	// If the user does not define the modeling name and modeling quota, it will be the default model.
+	// The default model index from 0 to 11. Each default model's cpu quota is (2^index-1, 2^index], index >= 0
```

Comment:
I don't think we really need so many models by default.
How about 6? Then, the maximum model is (32, 64)?

@kevin-wangzefeng How do you think?

Reply:
I also want to discuss this part. It is determined here, and the code of my implementation part should also be changed.

Reply:
Yes, the default model should be close to real production practice, thus most case users don't need to customize it.
@Poor12 is investigating it, let's wait for a while.

Reply:
// - index: 0
	// ranges:
	//   - name: CPU
	//     min: 0 C
	//     max: 1 C
	//   - name: memory
	//     min: 0 GB
	//     max: 4 GB
        // - index: 1
	// ranges:
	//   - name: CPU
	//     min: 1 C
	//     max: 2 C
	//   - name: memory
	//     min: 4 GB
	//     max: 16 GB
        // - index: 2
	// ranges:
	//   - name: CPU
	//     min: 2 C
	//     max: 4 C
	//   - name: memory
	//     min: 16 GB
	//     max: 32 GB
       // - index: 3
	// ranges:
	//   - name: CPU
	//     min: 4 C
	//     max: 8 C
	//   - name: memory
	//     min: 32 GB
	//     max: 64 GB
         // - index: 10
	// ranges:
	//   - name: CPU
	//     min: 64 C
	//     max: 128 C
	//   - name: memory
	//     min: 512 GB
	//     max: 1024 GB

Reply:
OK, I modify the comment in the API pr, then I will modify my Implementation code. I just change the default value. However, my test cases need to change a few groups case.